<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFGAppearance</class>
 <widget class="QWidget" name="CFGAppearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>392</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="cboLanguage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="translationRestartNotifierWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../../media/resources.qrc">:/icons/information-frame.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Restart will be required to apply the translation fully.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Player slots style:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="slotStyle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Marines</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blocks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Numeric</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Custom servers color:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ColorButton" name="btnCustomServersColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>LAN servers color:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ColorButton" name="btnLanServersColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Servers with buddies color:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="cbMarkServersWithBuddies">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorButton" name="btnBuddyServersColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bots are not players</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="cbBotsNotPlayers">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hide passwords</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="cbHidePasswords">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lookup server hosts</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="cbLookupHosts">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Colorize server console</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="cbColorizeConsole">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Draw grid in server table</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="cbDrawGridInServerTable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gboUseTrayIcon">
     <property name="title">
      <string>Use tray icon</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbCloseToTrayIcon">
        <property name="text">
         <string>When close button is pressed, minimize to tray icon.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>gui/widgets/colorbutton.h</header>
   <slots>
    <signal>colorUpdated(QColor,QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../media/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>slotStyle</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CFGAppearance</receiver>
   <slot>dynamicAppearanceChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDrawGridInServerTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>CFGAppearance</receiver>
   <slot>dynamicAppearanceChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCustomServersColor</sender>
   <signal>colorUpdated(QColor,QColor)</signal>
   <receiver>CFGAppearance</receiver>
   <slot>dynamicAppearanceChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLanServersColor</sender>
   <signal>colorUpdated(QColor,QColor)</signal>
   <receiver>CFGAppearance</receiver>
   <slot>dynamicAppearanceChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBuddyServersColor</sender>
   <signal>colorUpdated(QColor,QColor)</signal>
   <receiver>CFGAppearance</receiver>
   <slot>dynamicAppearanceChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMarkServersWithBuddies</sender>
   <signal>toggled(bool)</signal>
   <receiver>CFGAppearance</receiver>
   <slot>dynamicAppearanceChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboLanguage</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CFGAppearance</receiver>
   <slot>setVisibilityOfLanguageChangeNotificationIfNeeded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setVisibilityOfLanguageChangeNotificationIfNeeded()</slot>
 </slots>
</ui>
